### PR TITLE
Added support for scrobbling from DI.FM and FM4

### PR DESCRIPTION
**Pull request by Vlad1mir-D**
Original patch was based on https://github.com/zapster/mpdscribble-fm4 fork

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ config.h
 config.h.in
 mpdscribble
 .dirstamp
+objdir
```
Comment (MaxKellermann) [inline .gitignore]: What's this?

File: src/difm.c [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+#include <ctype.h>
+#include <assert.h>
+
+#include <glib.h>
+#include <mpd/tag.h>
+
+#define DIFM_THRESHOLD 10
+
+/*
+ * Hack libmpdclient
```
Comment (MaxKellermann) [inline src/difm.c]: This kind of hack is not acceptable because it may crash at any time.

File: .gitignore (hunk 1 above)
Comment (Am1GO) [reply .gitignore]: `objdir` is where I'm typically running `../configure && make -j ${SMP}`

File: src/difm.c (hunk 2 above)
Comment (Am1GO) [reply src/difm.c]: Do you mean detection thresholds or libmpdclient hacking?

File: .gitignore (hunk 1 above)
Comment (MaxKellermann) [reply .gitignore]: I don't think your personal choice of output directory needs to be part of the mpdscribble project, and it doesn't belong in this commit.

File: src/difm.c (hunk 2 above)
Comment (MaxKellermann) [reply src/difm.c]: I meant the line I was commenting on, i.e. the latter.

**Comment by MaxKellermann** [comment]
Issues were not fixed a year later.

**Pull request closed without merging**